projects/marketing-site/components/Team/CoreTeam.tsx

```diff
@@ -1,31 +1,30 @@
-import Image from "next/image";
-
-import distractionChadBunny from "@/images/chadbit.jpg";
-import dicaso from "@/images/dicaso.jpg";
-import weston from "@/images/pfer-weston.png";
-import commonLang from "@/lang/common.json";
+import distractionChadBunny from '@/images/chadbit.jpg';
+import dicaso from '@/images/dicaso.jpg';
+import jason from '@/images/jason-factor-avatar.png';
+import commonLang from '@/lang/common.json';
+import Image from 'next/image';
 import { useRouter } from 'next/router';
 
 const people = [
   {
-    name: "DistractionBoy",
-    role: "Engineering Lead",
+    name: 'DistractionBoy',
+    role: 'Engineering Lead',
     imageUrl: distractionChadBunny,
-    twitterUrl: "https://twitter.com/DistractionBoy_",
+    twitterUrl: 'https://twitter.com/DistractionBoy_',
     linkedinUrl: null,
   },
   {
-    name: "Dicaso",
-    role: "Creative Director",
+    name: 'Dicaso',
+    role: 'Creative Director',
     imageUrl: dicaso,
-    twitterUrl: "https://twitter.com/Dicaso5",
+    twitterUrl: 'https://twitter.com/Dicaso5',
     linkedinUrl: null,
   },
   {
-    name: "Weston Nelson",
-    role: "Treasury Management",
-    imageUrl: weston,
-    twitterUrl: "https://twitter.com/westonnelson",
```

Comment:
Can we use github for my profile ?. I'd like to show github than twitter. 
This is my github: https://github.com/anhfactor